src/main/java/hudson/plugins/analysis/collector/AnalysisPublisher.java

```diff
@@ -149,6 +68,21 @@ public boolean isCheckStyleActivated() {
     }
 
     /**
+     * Needed by workflow snippet generator.
+     * Having this ugly getter naming to keep backward configuration file compatibility on fields names.
+     * 
+     * @see #isCheckStyleActivated()
+     */
+    public boolean isIsCheckStyleActivated() {
```

Comment:
Also update the test scripts to match, of course.

BTW if there is any doubt about whether you set up some databinding fields correctly, it is wise to write a `configRoundTrip` test. For a freestyle publisher you can use some methods right on `JenkinsRule`. For use from Workflow (including `CoreStep`) you need `StepConfigTester`, a `tests`-classifier dependency.